woocommerce/woocommerce-ios
# Update GoogleSignIn to 6.0.1

Refs https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/609

This change updates `WordPressAuthenticator` dependency to `1.41.0-beta.2`, which upgrades `GoogleSignIn` to version `6.0.1`. Note that https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/610 needs to be merged first before merging this PR.

If this is too close to 7.3, feel free to let me know and I'll bump the milestone to 7.4 🙂 .

## To Test

- Ensure that the project compiles successfully.
- Ensure that Google sign-in flows work as expected. 

## Author Checklist

Update release notes:
